Doppler.Jobs.Server/Startup.cs

```diff
@@ -66,16 +64,26 @@ public void ConfigureServices(IServiceCollection services)
 
             var jobsConfig = new TimeZoneJobConfigurations
             {
-                TimeZoneJobs = $"{Configuration["TimeZoneJobs"]}"
+                TimeZoneJobs = Configuration["TimeZoneJobs"]
             };
 
             services.AddTransient(sp => new DopplerCurrencyService(
                 sp.GetService<IHttpClientFactory>(), 
                 httpClientPolicies,
                 dopplerCurrencySettings,
-                sp.GetService<ILoggerAdapter<DopplerCurrencyService>>(),
+                sp.GetService<ILogger<BaseExternalService>>(),
                 jobsConfig));
 
+            var dopplerSapServiceSettings = new DopplerSapServiceSettings();
+            Configuration.GetSection("DopplerSapService").Bind(dopplerSapServiceSettings);
+            services.AddSingleton(dopplerSapServiceSettings);
```

Comment:
If I am not wrong, you could use _IOptions_, maybe you can ask @VicThorX or @elsupergomez about it.

Reply:
Thanks, I resolved it in this commit https://github.com/MakingSense/doppler-jobs/pull/2/commits/24153e86346d2caf53371f24a469e6b5f355048b

Reply:
👏